opendbc/car/hyundai/fingerprints.py

```diff
@@ -1231,4 +1231,23 @@
       b'\xf1\x00T01G00BL  T01I00A1  DOS2T16X2XI00NS0\x8c`\xff\xe7',
     ],
   },
+  CAR.HYUNDAI_PALISADE_2023: {
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00LX2 MFC  AT USA LHD 1.00 1.04 99211-S8150 220622',
+      b'\xf1\x00ON  MFC  AT USA LHD 1.00 1.01 99211-S9150 220708',
+      b'\xf1\x00ON  MFC  AT USA LHD 1.00 1.00 99211-S9160 230303',
+      b'\xf1\x00ON  MFC  AT USA LHD 1.00 1.01 99211-S9160 230802',
+      b'\xf1\x00LX2 MFC  AT USA LHD 1.00 1.01 99211-S8600 230817',
+      b'\xf1\x00ON  MFC  AT USA LHD 1.00 1.00 99211-S9170 240531',
+    ],
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00LX2_ SCC -----      1.00 1.01 99110-S8150         ',
+      b'\xf1\x00ON__ SCC -----      1.00 1.01 99110-S9150         ',
+      b'\xf1\x00LX2_ SCC FHCUP      1.00 1.01 99110-S8150         ',
+      b'\xf1\x00ON__ SCC FHCUP      1.00 1.01 99110-S9150         ',
+      b'\xf1\x00ON__ SCC FHCUP      1.00 1.00 99110-S9160         ',
+      b'\xf1\x00LX2  SCC FHCUP      1.00 1.00 99110-S8600         ',
+      b'\xf1\x00ON__ SCC FHCUP      1.00 1.00 99110-S9170         ',
+    ],
+  },
```

Comment:
TODO: only the telluridee we tested, then we can add new FP as we do some standard torque checks, etc.